addons/curator-link/addon.json

```diff
@@ -1,16 +1,30 @@
 {
-  "name": "Front page curator link",
-  "description": "Makes the username of the front page curator a clickable link to their profile.",
+  "name": "Clickable usernames",
+  "description": "Makes the front page curator and mentions on the forums clickable links to their profile.",
   "credits": [
     {
       "name": "Hans5958",
       "link": "https://github.com/Hans5958/userscripts/blob/master/Scratch/Curator%20Links.user.js"
+    },
+    {
+      "name": "Samq64",
+      "link": "https://github.com/Samq64"
     }
   ],
   "userscripts": [
     {
       "url": "script.js",
-      "matches": ["https://scratch.mit.edu/"]
+      "matches": ["https://scratch.mit.edu/"],
+      "if": {
+        "settings": { "curator": true }
+      }
+    },
+    {
+      "url": "mentions.js",
+      "matches": ["topics"],
+      "if": {
+        "settings": { "mentions": true }
+      }
```

Comment:
See #5272

Have a single userscript that checks the settings when first ran. No need to support dynamically changing the settings - just 2 separate `if()` statements and that's all.